ansible/check_caching_service.yml

```
@@ -0,0 +1,20 @@
+#Issue setup ip address inside the hpa config file for postgres to accept the connection from it  
+- name: Check caching
+  hosts: "{{ idr_environment | default('idr') }}-management-hosts"
+
+  tasks:  
+  - name: Get infos on container
+    docker_container_info:
```

```suggestion
    become: yes
    docker_container_info:
```

I had to insert this to avoid a Permission denied